core/src/cluster_info.rs

```diff
@@ -1092,21 +1090,16 @@ impl ClusterInfo {
             .process_pull_request(caller, filter, now);
         let len = data.len();
         trace!("get updates since response {}", len);
-        if data.is_empty() {
-            trace!("no updates me {}", self_id);
-            vec![]
-        } else {
-            let rsp = Protocol::PullResponse(self_id, data);
-            // the remote side may not know his public IP:PORT, record what he looks like to us
-            //  this may or may not be correct for everybody but it's better than leaving him with
-            //  an unspecified address in our table
-            if from.gossip.ip().is_unspecified() {
-                inc_new_counter_info!("cluster_info-window-request-updates-unspec-gossip", 1);
-                from.gossip = *from_addr;
-            }
-            inc_new_counter_info!("cluster_info-pull_request-rsp", len);
-            to_shared_blob(rsp, from.gossip).ok().into_iter().collect()
+        let rsp = Protocol::PullResponse(self_id, data);
+        // the remote side may not know his public IP:PORT, record what he looks like to us
```

Comment:
nit: please reword this comment to remove all the masculine.  fullnodes have no gender :)

I know this is an existing comment but you touched it here so `git blame` will say it's yours now.